sdk/communication/Azure.Communication.PhoneNumbers/src/ClientDiagnostics.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json;
+using Azure.Communication.PhoneNumbers.Models;
+using Azure.Core;
+
+#nullable enable
```

Comment:
now that `<Nullable>enable</Nullable>` is removed from the csproj file, we should not explicitly disable here again

Reply:
Removed